qualcoder/view_graph.py

```diff
@@ -108,12 +110,12 @@ def get_first_with_attr(cats,**attrs):
             return x
 
 
-def plot_with_pygraphviz(cats,codes,topnode=None,prog='neato',rankdir='LR'):
-    import pygraphviz as pgv
-
+def plot_with_pygraphviz(cats,codes,codelinks,topnode=None,prog='neato',rankdir='LR'):
```

Comment:
it will take me some time to understand all the code here. but will accept that it is correct.